### DISCUSSION TITLE
Can't see tasks and some other objects with snowflake connection

### DISCUSSION THREAD

**Discussion by LmG7119** [dbeaver/dbeaver#34894] [General]
Am I missing something?

![SNFLK](https://github.com/user-attachments/assets/bef5ebaf-41e8-4fc5-abf5-c9ded3364e07)

**Comment by E1izabeth** [comment]
It's available in PRO versions. Here is documentation about it https://dbeaver.com/docs/dbeaver/Snowflake/#metadata
You can try trial version for free https://dbeaver.com/trial/
